### ISSUE TITLE
Applied Accessibility

### ISSUE THREAD

**Issue by itallosavieira** [itallosavieira/freeCodeCamp-Certifications#4]
- [x] Add a Text Alternative to Images for Visually Impaired Accessibility
- [x] Know When Alt Text Should be Left Blank
- [x] Use Headings to Show Hierarchical Relationships of Content
- [x] Jump Straight to the Content Using the main Element
- [x] Wrap Content in the article Element
- [x] Make Screen Reader Navigation Easier with the header Landmark
- [x] Make Screen Reader Navigation Easier with the nav Landmark
- [x] Make Screen Reader Navigation Easier with the footer Landmark
- [x] Improve Accessibility of Audio Content with the audio Element
- [x] Improve Chart Accessibility with the figure Element
- [x] Improve Form Field Accessibility with the label Element
- [x] Wrap Radio Buttons in a fieldset Element for Better Accessibility
- [x] Add an Accessible Date Picker
- [x] Standardize Times with the HTML5 datetime Attribute
- [x] Make Elements Only Visible to a Screen Reader by Using Custom CSS
- [x] Improve Readability with High Contrast Text
- [x] Avoid Colorblindness Issues by Using Sufficient Contrast
- [x] Avoid Colorblindness Issues by Carefully Choosing Colors that Convey Information
- [x] Give Links Meaning by Using Descriptive Link Text
- [x] Make Links Navigable with HTML Access Keys
- [x] Use tabindex to Add Keyboard Focus to an Element
- [x] Use tabindex to Specify the Order of Keyboard Focus for Several Elements
